test/e2e/metrics_runtime_input_resources_test.go

```diff
@@ -15,83 +15,84 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/kyma-project/telemetry-manager/internal/testutils"
+	"github.com/kyma-project/telemetry-manager/test/testkit/apiserverproxy"
 	"github.com/kyma-project/telemetry-manager/test/testkit/assert"
 	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
 	kitkyma "github.com/kyma-project/telemetry-manager/test/testkit/kyma"
 	. "github.com/kyma-project/telemetry-manager/test/testkit/matchers/metric"
 	"github.com/kyma-project/telemetry-manager/test/testkit/metrics/runtime"
 	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
 	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/prommetricgen"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/telemetrygen"
 	"github.com/kyma-project/telemetry-manager/test/testkit/periodic"
 	"github.com/kyma-project/telemetry-manager/test/testkit/suite"
 )
 
 var _ = Describe(suite.ID(), Label(suite.LabelMetrics), Label(suite.LabelSetA), Ordered, func() {
-	Context("When metric pipelines with container, pod and node metrics enabled exist", Ordered, func() {
+	Context("When metric pipelines with input resources exist", Ordered, func() {
 		var (
-			mockNs = suite.IDWithSuffix("container-pod-node-metrics")
+			mockNs = suite.IDWithSuffix("resource-metrics")
```

Comment:
Since we have now only one namespace created in this test suite, we don't need to add any suffix to its name. It is enough now to use `suite.ID()`